### PR TITLE
fix: Correct upstream podman issue

### PR DESCRIPTION
**Pull request by KyleGospo**

### CODE REVIEW

**Review by p5 — APPROVED** [review]

**Review by castrojo — APPROVED** [review]

File: Containerfile [hunk 1]
```diff
@@ -44,6 +44,9 @@ COPY kmods-sys_files /tmp/kmods-files
 
 COPY --from=ghcr.io/ublue-os/akmods:main-${FEDORA_MAJOR_VERSION} /rpms /tmp/akmods-rpms
 
+# Workaround for podman issue upstream.
+RUN rpm-ostree override replace https://bodhi.fedoraproject.org/updates/FEDORA-2023-00c78aad58
```
Comment (p5) [inline Containerfile:48]: Maybe a stupid question, but will this install the F39 version of Podman on F38 images too?

Comment (castrojo) [reply Containerfile:48]: I think we'd want to pin it on 38 too, though I suspect they'll fix the issue upstream before upgrading 38 anyway.

**Comment by p5** [comment]
Incorrect commit message, but this should fix the issues hopefully

File: Containerfile [hunk 2]
```diff
@@ -19,6 +19,13 @@ COPY github-release-install.sh \
 COPY --from=ghcr.io/ublue-os/config:latest /rpms /tmp/rpms
 COPY --from=ghcr.io/ublue-os/akmods:main-${FEDORA_MAJOR_VERSION} /rpms/ublue-os /tmp/rpms
 
+# # Workaround for podman issue upstream - https://github.com/containers/podman/issues/20872
+RUN if [ "$FEDORA_MAJOR_VERSION" = "39" ]; then \
+        rpm-ostree override replace https://bodhi.fedoraproject.org/updates/FEDORA-2023-00c78aad58; \
+    elif [ "$FEDORA_MAJOR_VERSION" = "38" ]; then \
+        rpm-ostree override replace https://bodhi.fedoraproject.org/updates/FEDORA-2023-c72015807c; \
+    fi
```
Comment (p5) [inline Containerfile:27]:
```suggestion
        rpm-ostree override replace https://bodhi.fedoraproject.org/updates/FEDORA-2023-00c78aad58; \
    fi
```

**Pull request merged**